Various technical notes/transition_infos.json

```
@@ -960,20 +927,96 @@
           "area_id": "0x62548B77",
           "area_name": "White Valley",
           "requires": null
-        },
-        {
-          "area_id": "0xDEDA69BC",
-          "area_name": "Jungle Canyon",
-          "requires": [
-            [
-              "GLITCHES"
-            ],
-            [
-              "pickaxe"
-            ]
-          ]
         }
```

1-way cavern lake exit

```suggestion
        },
        {
          "area_id": "0xDEDA69BC",
          "area_name": "Jungle Canyon",
          "requires": [
            [
              "GLITCHES"
            ],
            [
              "pickaxe"
            ]
          ]
        }
```

---

This is one of the one-way-exits. All one-way-exits are removed from the .json file and instead handled in the entrance_rando.py file.

---

I mark this as resolved because it's a duplicate of another conversation.